requirements.txt

```diff
@@ -1,3 +0,0 @@
-requests
-feedparser
-Pillow
```

Comment:
I'd prefer if this file was left in the repo. The video mentions a different way to start this program, so it might confuse people if they glaze over the documentation.